internal/bundlereader/loaddirectory.go

```diff
@@ -151,10 +152,10 @@ func readFleetIgnore(path string) ([]string, error) {
 	return ignored, nil
 }
 
-func loadDirectory(ctx context.Context, compress bool, prefix, base, source, version string, auth Auth) ([]fleet.BundleResource, error) {
+func loadDirectory(ctx context.Context, compress bool, disableDependencyUpdate bool, prefix, base, source, version string, auth Auth) ([]fleet.BundleResource, error) {
```

Comment:
yeah, I discussed the name with Mario as well, because I thought it was too large, too.
We said to keep it and, as it refers right away to the `fleet.yaml` option, I guess it's fine to keep the same name.
No strong opinion here, if you do... I can change the parameter name in all functions.